packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -1262,7 +1262,7 @@ const schema: SchemaType<DbPost> = {
             false,
           );
         }
-        return data.frontpageDate ?? oldDocument.frontpageDate;
+        return data.frontpageDate === undefined ? oldDocument.frontpageDate : data.frontpageDate;
```

Comment:
Fuuuuuuuck this language bro. Can you add a comment please, there's no way a future dev is gonna intuit this.